src/app/system/configurations/global-configurations-tab/clone-configuration/clone-configuration.component.html

```diff
@@ -14,6 +14,11 @@ <h1>Clone Configuration</h1>
                 Value is <strong>required</strong>
               </mat-error>
             </mat-form-field>
+
```

Comment:
@william000000 Willy this add the String Value to the clone page for all configuration? And also please update the release notes